### PR TITLE
lit-af fix for multiple lit instances on single computer.

### PR DESCRIPTION
**Pull request by infografx**
Now you can run the lit-af for several instances of a lit on single machine as:

./lit-af --litHomeDir=/path/to/lit1 --autoListenPort=2448
./lit-af --litHomeDir=/path/to/lit2 --autoListenPort=2449

### CODE REVIEW

File: WALKTHROUGH.md [hunk 1]
```diff
@@ -57,36 +57,56 @@ Now in another window, alice connects to the lit node over RPC using `./lit-af`
...
-Alice sets her node to listen:
+ls -a shows how much money Alice has, which is none.  She can get some from a faucet. One such faucet you can find [here](https://testnet.manu.backend.hamburg/faucet).
 
-```
-lit-af# lis
-entered command: lis
-listening on ln1pmclh89haeswrw0unf8awuyqeu4t2uell58nea@:2448
-```
+Bob can start a wallet in the same way; if Bob's node is running on the same computer, he'll have to run lit placing a lit.conf in to the ~/bnode folder:
+
```
Comment (Varunram) [inline WALKTHROUGH.md]: Would be nice if you could enclose them with reverse ticks like
```
sample ticks: ```
```
thanks!

File: cmd/lit-af/lit-af.go [hunk 2]
```diff
@@ -68,17 +71,20 @@ func newConfigParser(conf *litAfConfig, options flags.Options) *flags.Parser {
...
-	_, err = os.Stat(defaultDir)
-	if os.IsNotExist(err) {
-		os.Mkdir(defaultDir, 0700)
-	}
+
+	//----------------
 
 	preParser := newConfigParser(&conf, flags.HelpFlag)
+
+	//----------------
```
Comment (Varunram) [inline cmd/lit-af/lit-af.go]: Please remove these empty line comments

File: cmd/lit-af/lit-af.go [hunk 3]
```diff
@@ -89,20 +95,45 @@ func (lc *litAfClient) litAfSetup(conf litAfConfig) {
...
+	}
+
+	//-----------------------------
+
 	adr, host, port := lnutil.ParseAdrStringWithPort(conf.Con)
-	logging.Infof("Adr: %s, Host: %s, Port: %d", adr, host, port)
-	if litrpc.LndcRpcCanConnectLocallyWithHomeDir(defaultDir) && adr == "" && (host == "localhost" || host == "127.0.0.1") {
 
-		lc.RPCClient, err = litrpc.NewLocalLndcRpcClientWithHomeDirAndPort(defaultDir, port)
+	//-----------------------------
```
Comment (Varunram) [inline cmd/lit-af/lit-af.go]: Again, refer top comments, lets not have these one line comments like:
```
--------
```

**Comment by infografx** [comment]
I took into account the comments.

**Comment by adiabat** [comment]
Cool - looks good to me, thanks!

**Comment by infografx** [comment]
Test.

**Pull request merged**